bigtable-dataflow-parent/bigtable-hbase-beam/pom.xml

```diff
@@ -165,48 +165,6 @@ limitations under the License.
         </plugins>
       </build>
     </profile>
-    <profile>
-      <id>release</id>
-      <build>
-        <plugins>
-          <plugin>
-            <groupId>org.apache.maven.plugins</groupId>
-            <artifactId>maven-javadoc-plugin</artifactId>
```

Comment:
I dont think you can just remove this...the javadoc config needs to exist somewhere. Maybe move it to pluginManagement?

Reply:
Thank you, meant to ask about this. I'm thinking I should model it to how java-bigtable does javadoc, given they will eventually have the same parent and configuration? https://github.com/googleapis/java-bigtable/blob/master/pom.xml#L215

Reply:
I think moving the config to the root pom is fine, but please add that in this PR.
When you say same parent & configuration, do you mean google-cloud-shared-config?

Reply:
Sorry for being unclear, I simply meant the structure - I'm planning on leaving the configs in the same pom files. And yes to question 2